panel/chat/message.py

```diff
@@ -630,6 +626,21 @@ def _update_avatar_pane(self, event=None):
             del params['name']
             self._left_col[0].param.update(**params)
 
+    def _render_reaction_icons(self):
+        reaction_icons = self.reaction_icons
```

Comment:
Seems like if we don't allow a dict to be passed in dynamically we will have no way to provide reaction_icons to a ChatFeed since we can't pass an instance there.

Reply:
I've always wondered about this:
https://discourse.holoviz.org/t/is-there-a-way-to-validate-a-parameter-in-param/7196

But I believe we don't need to dynamically pass a dict because ChatFeed creates new messages (if I'm not mistaken)